pypeit/scripts/setup.py

```diff
@@ -43,7 +43,7 @@ def get_parser(cls, width=None):
                             help='Include the background-pair columns for the user to edit')
         parser.add_argument('-m', '--manual_extraction', default=False, action='store_true',
                             help='Include the manual extraction column for the user to edit')
-        parser.add_argument('-v', '--verbosity', type=int, default=2,
+        parser.add_argument('-v', '--verbosity', type=int, default=1,
```

Comment:
Is there a reason to use the level=1 verbosity instead of 2?

Reply:
All scripts other than `run_pypeit` have a default verbosity setting of 1.  Also, since this script didn't have the `msgs.set_logfile_and_verbosity()` call yet, it was running with a verbosity of 1 all the time regardless of what the user requested.

Reply:
Ah, good point.